src/libpanic_unwind/seh.rs

```diff
@@ -255,6 +296,12 @@ pub unsafe fn panic(data: Box<dyn Any + Send>) -> u32 {
         &mut CATCHABLE_TYPE.pType as *mut _ as *mut u32,
         ptr!(&TYPE_DESCRIPTOR as *const _) as u32,
     );
+    if !cfg!(bootstrap) {
+        atomic_store(
+            &mut CATCHABLE_TYPE.copyFunction as *mut _ as *mut u32,
+            ptr!(exception_copy) as u32,
+        );
+    }
```

Comment:
If we don't define the copyFunction, is it implicitly assumed that `memcpy` "just works" by the caller? I think I read something to that effect while looking into this a couple days back, but I can't find it right now.

Reply:
Yes, a NULL `copyFunction` causes the exception to be copied using `memcpy`. This is what we previously did, but this could lead to a double-free if the exception is copied and thrown twice (which you can do in C++ by capturing an exception with `exception_ptr`).